database-commons/src/main/java/io/cdap/plugin/db/source/AbstractDBSource.java

```diff
@@ -373,6 +373,12 @@ protected Class<? extends DBWritable> getDBRecordType() {
     return DBRecord.class;
   }
 
+  /**
+   * Returns the external documentation link.
+   * Override this method to provide a custom external documentation link.
+   *
+   * @return external documentation link
+   */
   protected String getExternalDocumentationLink() {
```

Comment:
I think we should make this `abstract` and enforce the child to implement it correctly.

Returning null might cause issue if someone misses to override this.